Examples/Scripts/Python/simulate_barrel.py

```
@@ -0,0 +1,90 @@
+def main():
+    from argparse import ArgumentParser
+
+    u = acts.UnitConstants
+ 
+    parser = ArgumentParser()
+    parser.add_argument("-i", "--input", type=str, default="/eos/user/c/cimuonsw/GeometryFiles/MockUp.db", help="Input SQL file")
+    parser.add_argument("--outDir", default="./", help="Output" )
+
+    args = parser.parse_args()
+
+    gContext = acts.GeometryContext()
```

Ruff (0.11.9)

Local variable `gContext` is assigned to but never used

Remove assignment to unused variable `gContext`

(F841)